src/test/java/org/sagebionetworks/bridge/spring/filters/MetricsFilterTest.java

```diff
@@ -62,7 +61,8 @@ public void metricsTest() throws Exception {
         when(mockRequest.getRemoteAddr()).thenReturn(TestConstants.IP_ADDRESS);
         when(mockRequest.getHeader(USER_AGENT)).thenReturn("userAgent/1");
         when(mockResponse.getStatus()).thenReturn(201);
-        
+        when(mockRequest.getQueryString()).thenReturn("consents=true&email=should_not_leak@fake.com");
```

Comment:
Consider adding query params with %20 and + to make sure we're calling the right URL Decode method.